<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="customItemsGroupBox">
         <property name="title">
          <string>Custom Items</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="customItemLabel">
            <property name="text">
             <string>These items are always pinned at the top of the root menu</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="customItemsHorizontalLayout">
            <item>
             <widget class="QTreeWidget" name="customItemTree">
              <property name="indentation">
               <number>0</number>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="customItemsVerticalLayout">
              <item>
               <widget class="QToolButton" name="addCustomItemButton">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/res/icons/add.png</normaloff>:/res/icons/add.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="deleteCustomItemButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/res/icons/bin.png</normaloff>:/res/icons/bin.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="upCustomItemButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/res/icons/arrow_up.png</normaloff>:/res/icons/arrow_up.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="downCustomItemButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/res/icons/arrow_down.png</normaloff>:/res/icons/arrow_down.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="customItemsVerticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="hiddenVolumesGroupBox">
         <property name="title">
          <string>Hide Volumes</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="hiddenVolumesLabel">
            <property name="text">
             <string>Select the volumes you want hidden</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="hiddenVolumeTree">
            <property name="indentation">
             <number>0</number>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="hiddenVolumesVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aboutTab">
      <attribute name="title">
       <string>About</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
